### PR TITLE
:wrench: chore(api): add additional troubleshooting instructions to publish api docs

### PR DESCRIPTION
**Pull request by iamrajjoshi**
image

i added a new type of error that i have ran into and some instructions on fix

### CODE REVIEW

File: develop-docs/backend/api/public.mdx [hunk 1]
```diff
@@ -481,6 +481,28 @@ NOTE: The `openapi-diff` test is supposed to fail when CI runs on your pull requ
 
 **Solution**: You need to add the endpoint to a [sidebar tab](/backend/api/public/#3-sidebar-tab) with the decorator `@extend_schema(tags=['<section title>'])`.
```
Comment (isabellaenriquez) [inline develop-docs/backend/api/public.mdx:483]:
```suggestion

---

```
spacing nit :)

File: develop-docs/backend/api/public.mdx [hunk 2]
```diff
@@ -481,6 +481,28 @@ NOTE: The `openapi-diff` test is supposed to fail when CI runs on your pull requ
...
 
+**Problem**: `drf_spectacular.plumbing.UnableToProceedError' <class 'serializer_parth.FooSerializer'> ... raise UnableToProceedError(hint)`
+
+**Solution**: Check that the response of your API documentation is using a TypedDict rather than a serializer.
+
+If the scheme looks something like this:
```
Comment (isabellaenriquez) [inline develop-docs/backend/api/public.mdx:488]:
```suggestion
If the schema looks something like this:
```

File: develop-docs/backend/api/public.mdx [hunk 3]
```diff
@@ -481,6 +481,28 @@ NOTE: The `openapi-diff` test is supposed to fail when CI runs on your pull requ
 
 **Solution**: You need to add the endpoint to a [sidebar tab](/backend/api/public/#3-sidebar-tab) with the decorator `@extend_schema(tags=['<section title>'])`.
 
+**Problem**: `drf_spectacular.plumbing.UnableToProceedError' <class 'serializer_parth.FooSerializer'> ... raise UnableToProceedError(hint)`
```
Comment (isabellaenriquez) [inline develop-docs/backend/api/public.mdx:484]:
```suggestion
**Problem**: `drf_spectacular.plumbing.UnableToProceedError' <class 'serializer_path.FooSerializer'> ... raise UnableToProceedError(hint)`
```

**Review by ameliahsu — APPROVED** [review]
thanks 😸

**Review by isabellaenriquez — APPROVED** [review]
nits, ty for updating!

**Pull request merged**